index.md

```diff
@@ -36,3 +36,29 @@ helm install <your-relase-name> cstor/openebs-cstor --create-namespace
 ```bash
 helm install <your-relase-name> cstor/openebs-cstor
 ```
+
+#### Upgrade OpenEBS CStor
```

Comment:
@shubham14bajpai Should the data plane update steps also be given here?

Reply:
We can add a upgrade repo doc link ?

Reply:
We can link this doc here https://github.com/openebs/upgrade/blob/master/docs/upgrade.md

Reply:
Done.